spec/react-masonry-component-test.js

```diff
@@ -101,8 +103,8 @@ describe('React Masonry Component', function() {
     });
 
     it('should allow custom props', function() {
-        const handler = () => {};
-        const component = TestUtils.renderIntoDocument(<MasonryComponent onClick={handler} test="testProp"/>);
+        const handler = function() {};
+        const component = TestUtils.renderIntoDocument(<MasonryComponent onClick={handler}/>);
```

Comment:
hmm, the more I think about it the more it makes no sense to even have this test. I think we can safely delete it.